utils/duplicate/loader.py

```diff
@@ -0,0 +1,16 @@
+from hca.loader import HcaLoader
+from .submission import DuplicateSubmission
+
+
+class DuplicateLoader(HcaLoader):
+    def duplicate_project(self, project_uuid: str, submission: DuplicateSubmission = None) -> DuplicateSubmission:
+        if not submission:
+            submission = DuplicateSubmission()
```

Comment:
I'm a bit confused by this. If there's no submission given, it creates one?

Reply:
Yep, the submission in this case is just a container for linked metadata.
This implementation allows the Duplicate Loader to be used for both the use cases in archiver.py

1. Load an entire project, and 
2. Load a single BioSample and Project.